src/components/indicators/indicators.js

```diff
@@ -15,34 +16,20 @@ define(['datetime', 'itemHelper', 'emby-progressbar', 'css!./indicators.css', 'm
         return false;
     }
 
-    function getProgressHtml(pct, options) {
-        var containerClass = 'itemProgressBar';
-        if (options) {
-            if (options.containerClass) {
-                containerClass += ' ' + options.containerClass;
-            }
-        }
+    export function getProgressHtml(pct, options) {
+        var containerClass = options && options.containerClass ? 'itemProgressBar ' + options.containerClass : 'itemProgressBar';
```

Comment:
I think that there is no need to minify so hard. Just merge conditional blocks, and it will be more readable.

Reply:
I agree, I find it harder to read like this than how it was before.

No need to try to write everything as short as possible, Babel and Terser are there already to sort that out for us.

Reply:
Its just me i personally prefer ternary more on such cases. Should i remove this from here?

Reply:
I like ternaries, but opinions are somewhat divided in my experience. Regarding the `let` and `const` usage, I'd like to remind everyone that incremental improvements are best. If this pull request doesn't already use ES6 declarations I don't believe that should be blocking at all.